### DISCUSSION TITLE
Request to deploy slash commands automatically instead of deploying manually...

### DISCUSSION THREAD

**Discussion by testbot-github** [SudhanPlayz/Discord-MusicBot#654] [General]
For now in v5, we need to deploy slash commands manually.

The issue is many people who are testing v5 rn will tell that - `Slash commands not working...`

**So the ideal way to fix this is to deploy slash commands automatically.**

**Comment by DarrenOfficial** [comment]
No.

**Comment by DarrenOfficial** [comment]
v5 isn't ready.
